crates/runtime/src/embeddings/vector_search.rs

```diff
@@ -1478,7 +1482,7 @@ pub(crate) mod tests {
         let average_time = total_time / (timings.len() as u32);
         let average_time_ns = average_time.as_nanos();
         assert!(
-            average_time_ns < 1_000_000,
+            average_time_ns < 1_250_000,
```

Comment:
I think experience has shown that these types of tests are not reliable enough - is there another way we can test this?